### PR TITLE
Add note to convert ported docstrings

### PR DESCRIPTION
**Pull request by asmacdo**
https://pulp.plan.io/issues/2347

### CODE REVIEW

File: docs/contributing/3.0-development/index.rst [hunk 1]
```diff
@@ -24,4 +24,15 @@ please file an issue in Redmine (related to https://pulp.plan.io/issues/2247) to
 track the conversion of the dependency to support Python 3 and begin working with
 upstream to convert the package.
 
+Docstrings
+----------
 
+PUP-2  TODO(asmacdo)link adopted the use of Google Style Docstrings TODO(asmacdo)link. When developers port code from Pulp 2 to Pulp 3, they should also convert all the docstrings to the new style.
```
Comment (bmbouter) [inline docs/contributing/3.0-development/index.rst]: I went to review this, but it looks like it is still a work in progress. We shouldn't commit TODOs to the docs so maybe put a WIP? I can review whenever if you put a comment w/ my nick on it.

Comment (asmacdo) [inline docs/contributing/3.0-development/index.rst]: I missed that change in git. The corrected links are pushed.

**Comment by asmacdo** [comment]
@bmbouter ready for review

File: docs/contributing/3.0-development/index.rst [hunk 2]
```diff
@@ -24,4 +24,13 @@ please file an issue in Redmine (related to https://pulp.plan.io/issues/2247) to
 track the conversion of the dependency to support Python 3 and begin working with
 upstream to convert the package.
 
+Docstrings
+----------
 
+ `PUP-2 <https://github.com/pulp/pups/blob/master/pup-0002.md>`_ adopted Google style for :ref:`google-docstrings`. When developers port code from Pulp 2 to Pulp 3, they should also convert all the docstrings to the new style. vim-style regexes can be used to speed up the process. Together, these will convert all of the parameters to Google Style::
```
Comment (werwty) [inline docs/contributing/3.0-development/index.rst]: extra space at the beginning of this line is causing styling issue

Comment (werwty) [inline docs/contributing/3.0-development/index.rst]: s/developers port/porting/

Comment (werwty) [inline docs/contributing/3.0-development/index.rst]: s/they should//

**Pull request merged**